patches/@truffle+contract-schema+3.3.2.patch

```diff
@@ -1,369 +0,0 @@
-diff --git a/node_modules/@truffle/contract-schema/index.js b/node_modules/@truffle/contract-schema/index.js
```

Comment:
I'm not sure we can remove this patch otherwise, we won't be able to change the openeth/truffle-typings. Maybe we could do this in a new package but eventually, this should be handled in a different task if this was the idea

Reply:
In commons we dont use this library and its not installed, therefore the patch application failed and thats why I removed it.

Reply:
I suppose this is still valid, isn't it? In my understanding, we should always have the patches, in each repo: am I missing anything?

Reply:
yes we should, but the patches are there